src/controllers/applicationsContentController.js

```
@@ -43,6 +43,208 @@ const templateResponseValue = async (pageName) => {
+            {
+                keys: keyManager.sandboxKeys || {},
+                keyType: 'SANDBOX'
+            }
+        ];
+    });
+
+    let subscriptionScopes = [];
+    if (applicationReference) {
+        let cpApplication = await getAPIMApplication(req, applicationReference);
+        if (cpApplication && Array.isArray(cpApplication?.subscriptionScopes)) {
+            for (const scope of cpApplication?.subscriptionScopes) {
```

Biome (2.1.2)

[error] Unsafe usage of optional chaining.

If it short-circuits with 'undefined' the evaluation will throw TypeError here:


(lint/correctness/noUnsafeOptionalChaining)